src/docs/Components/Form/Select/index.mdx

```diff
@@ -4,13 +4,15 @@ order: 9
 description: 'A select is a selection tool used in forms, allowing user to choose between multiple predefined options.'
 links:
   vue:
-    link: 'https://adeo.github.io/mozaic-vue/?path=/story/components-mselect--try-it'
+    link: 'https://adeo.github.io/mozaic-vue/?path=/story/components-mselect'
```

Comment:
The link is broken.
The good url is => https://adeo.github.io/mozaic-vue/?path=/story/components-form-mselect--default